modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -354,6 +392,43 @@ struct IEUnit {
     }
 };
 
+bool IEUnit::InputFramesDesc::is_applicable(const cv::GMetaArg &mm) {
```

Comment:
IMHO
```
return `cv::holds_alternative<cv::GFrameDesc>`
```

BTW, why don't just check directly in kernel code instead of implementing 1-line method.
Probably the answer is that it can be expanded in future, but I don't get how...

BTW, `GFrameDesc` with fmt == `BGR` isn't applicable now